### PR TITLE
Add Parameters. Fixes #95

### LINKED ISSUE #95: Parameters for Sponge Commands
Following on from #89, the parameters of the new Sponge commands need to be added to the Table at source/users/commands.rst. Details and some examples are found at https://github.com/SpongePowered/Sponge/commit/be08be04b06b413686c7f6c574bf89e918e1d6d3
The examples would be useful to add to the page at users/config instead.
(A new page for Sponge commands may be justified if it gets much bigger.)

### PR DESCRIPTION
**Pull request by BlackGoat**
Won't fit in table without being ugly

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/4b4a8f07/index.html.

Here are some links to the pages that were modified:

- modified https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/4b4a8f07/server/spongineer/commands.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/210#issuecomment-100625519), not to the direct url to the preview._

File: source/server/spongineer/commands.rst [hunk 1]
```diff
@@ -38,6 +38,33 @@ Command               Description                               Permission
...
+* /sponge save [-g] [-d dim|*] [-w world|*]
+* /sponge reload [-g] [-d dim|*] [-w world|*]
+
+
+.. tip::
+
+    Here are a few simple examples of the sponge conf command in action. Please see
+    :doc:`../getting-started/configuration/index` for a more detailed explanation.
+
+    a. ``/sponge conf -d logging.chunk-load true``
```
Comment (boformer) [inline source/server/spongineer/commands.rst]: Remove the -d

File: source/server/spongineer/commands.rst [hunk 2]
```diff
@@ -38,6 +38,33 @@ Command               Description                               Permission
...
+     
+     Since no dimension was specified, the dimension would default to the sender(player) dimension. So if you were in a mystcraft dimension, this would alter the mystcraft dimension config.
+
+    b. ``/sponge conf -d nether logging.chunk-load true``
+     
+     Since a dimension type was specified, this would alter the nether dimension config.
```
Comment (boformer) [inline source/server/spongineer/commands.rst]: Maybe a note that this would affect all nether dimensions.

**Comment by BlackGoat** [comment]
I have no idea why or how I broke the table :/

**Pull request closed without merging**